website/src/pages/Compare/Compare.jsx

```diff
@@ -0,0 +1,272 @@
+                    
+                    const jsonDataMicrobench = await responseMicrobench.json();
+                    console.log(jsonDataMicrobench)
+                    setDataMicrobench(jsonDataMicrobench)
+                } catch (error) {
+                    console.log('Error while retrieving data from the API', error);
+                    setError(errorApi);
+                }
+            };
+            fetchData();
+         
+    }, [gitRefLeft, gitRefRight])
+
+    // Changing the URL relative to the reference of a selected benchmark.
+    // Storing the carousel position as a URL parameter.
+    const navigate = useNavigate();
+    
+    useEffect(() => {
+        navigate(`?ltag=${gitRefLeft}&rtag=${gitRefRight}&ptag=${currentSlideIndex}&ptagM=${currentSlideIndexMobile}`)
+    }, [gitRefLeft, gitRefRight, currentSlideIndex, currentSlideIndexMobile]) 
+    
+    const handleInputChangeLeft = (e) => {
+        setGitRefLeft(e.target.value);
+      };
+    
+
+      const handleInputChangeRight = (e) => {
+        setGitRefRight(e.target.value);
+      };
+
+      const handleSlideChange = (swiper) => {
+        setCurrentSlideIndex(swiper.realIndex);
+    };
+
+    const slicedRef = gitRefLeft.slice(0, 8);
+    return (
+        <div className='compare'>
+            <div className='compare__top justify--content'>
+                <div className='compare__top__text'>
+                    <h2>Compare</h2>
+                    <span>
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer a augue mi.
+                            Etiam sed imperdiet ligula, vel elementum velit.
+                            Phasellus sodales felis eu condimentum convallis.
+                            Suspendisse sodales malesuada iaculis. Mauris molestie placerat ex non malesuada.
+                            Curabitur eget sagittis eros. Aliquam aliquam sem non tincidunt volutpat. 
+                    </span>
+                    
+                </div>
+                <figure className='compareStats'></figure>
+            </div>
+            <figure className='line'></figure>
+            <div className='compare__bottom'>
+                <div className='justify--content form__container'>
+                    <h3>Comparing <a href={`https://github.com/vitessio/vitess/commit/${gitRefLeft}`}>{gitRefLeft.slice(0, 8)}</a> with <a href={`https://github.com/vitessio/vitess/commit/${gitRefRight}`}>{gitRefRight.slice(0, 8)}</a></h3>
+                    <form className='justify--content'>
+                        <input
+                        type="text"
+                        value={gitRefLeft}
+                        onChange={handleInputChangeLeft}
+                        placeholder="Left commit SHA"
+                        className='form__inputLeft'></input>
```

Comment:
For both inputs we should have only a place holder, the field should be empty. It currently has `Left` and `Right` that we need to remove before being able to type code.